### ISSUE TITLE
Amount of RPCs more than 19 cause error - protoc-gen-twirp_ts: Plugin output is unparseable

### ISSUE THREAD

**Issue by f0mster** [hopin-team/twirp-ts#18]
If you'll try to run generation on this file - you will get an error.
If you'll delete one of them - all will work fine

```
syntax = "proto3";

service authentication {
  rpc A1(Am) returns (Am);
  rpc A2(Am) returns (Am);
  rpc A3(Am) returns (Am);
  rpc A4(Am) returns (Am);
  rpc A5(Am) returns (Am);
  rpc A6(Am) returns (Am);
  rpc A7(Am) returns (Am);
  rpc A8(Am) returns (Am);
  rpc A9(Am) returns (Am);
  rpc A10(Am) returns (Am);
  rpc A11(Am) returns (Am);
  rpc A12(Am) returns (Am);
  rpc A13(Am) returns (Am);
  rpc A14(Am) returns (Am);
  rpc A15(Am) returns (Am);
  rpc A16(Am) returns (Am);
  rpc A17(Am) returns (Am);
  rpc A18(Am) returns (Am);
  rpc A19(Am) returns (Am);
  rpc A20(Am) returns (Am);
}

message Am {}
```

**Comment by fenos** [comment]
@f0mster thanks for reporting this, 

I will investigate, I suspect is something to do with the buffer size of the output. 

Will keep you posted

**Comment by fenos** [comment]
Right! This one was a hard one!

To give some context, 
`protoc` use the `stdout` output to grab the information to generate a file from a plugin.

NodeJS by default is buffering data to `stdout` which means that for bigger files it will split the content into multiple chunks and output them async (to not block the event loop)

This is a problem for `protoc` as it is grabbing the first `stdout` chunk and try to generate the file, of course, the chunk is incomplete and it fails to parse the information.

The only way I found to fix this, is to instruct node to block for `stdout` calls using a private internal function:

```
process.stdout._handle.setBlocking(true);
```

The above line will prevent node to buffer to the `stdout` and `protoc` is happy.

This fix should probably live in the `@protobuf-ts/plugin-framework` as this issue could be common for any other files that are bigger than the buffered chunk.

Either way, I'll be shipping a fix for this ASAP

in the meantime you can amend `./node_modules/.bin/protoc-gen-twirp_ts` with the following:

```
#!/usr/bin/env node
process.stdout._handle.setBlocking(true);
require('./build/plugin')
```

**Comment by fenos** [comment]
the [PR](https://github.com/timostamm/protobuf-ts/pull/135) is up to fix this in the `plugin-framework` once that is merged, the fix will land on twirp-ts too

**Comment by fenos** [comment]
This fix is now available in `v2.3.1`

Since it was 1 liner,
I've just added it to the `./node_modules/.bin/protoc-gen-twirp_ts` entry point.
The main PR might take a while to get into the plugin framework with a new release.

Once that PR is merged I'll just remove my temp fix as those 2 lines will be added in the framework.

Thanks for reporting this and have a good day!

**Comment by mcfedr** [comment]
Great to have a really quick fix for this thanks! I came here looking for this issue I think about an hour after @f0mster had reported it yesterday!
